By default, the ASAN message goes to STDERR. This causes problem if we cannot
log the ASAN message, and we can not figure out why the ASAN check fails.
This change set the ASAN message goes to /va/log/asan${pid} by default.
--- llvm/projects/compiler-rt/lib/asan/asan_rtl.cc	2014-02-07 10:39:32.691683530 -0800
+++ llvm/projects/compiler-rt/lib/asan/asan_rtl.cc	2014-02-07 10:41:01.652390800 -0800
@@ -79,6 +79,9 @@
 }
 
 static const char *MaybeUseAsanDefaultOptionsCompileDefiniton() {
+#ifndef ASAN_DEFAULT_OPTIONS
+#define ASAN_DEFAULT_OPTIONS log_path=/var/log/asan
+#endif
 #ifdef ASAN_DEFAULT_OPTIONS
 // Stringize the macro value.
 # define ASAN_STRINGIZE(x) #x
